IncomingCallRouting/IncomingCallRouting/Utils/CallConfiguration.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Extensions.Configuration;
+
+namespace IncomingCallRouting
+{
+    /// <summary>
+    /// Configuration assoociated with the call.
+    /// </summary>
+    public class CallConfiguration
+    {
+        public static CallConfiguration callConfiguration = null;
+        public CallConfiguration(string connectionString, string appBaseUrl, string audioFileUri, string participant)
+        {
+            this.ConnectionString = connectionString;
+            this.AppBaseUrl = appBaseUrl;
+            this.AudioFileName = audioFileUri;
```

Comment:
Any limitation we used audiofilename and putting URI into it? can be directly set as AudioFileUri.